ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -0,0 +1,153 @@
+    'ceph_osd_getmaxosd',
+    'ceph_osd_lspools',
+    'ceph_osd_numa-status',
+    'ceph_osd_perf',
+    'ceph_osd_pool_ls_detail',
+    'ceph_osd_stat',
+    'ceph_osd_tree',
+    'ceph_osd_utilization',
+    'ceph_pg_dump',
+    'ceph_pg_stat',
+    'ceph_progress',
+    'ceph_quorum_status',
+    'ceph_report',
+    'ceph_service_dump',
+    'ceph_status',
+    'ceph_time-sync-status',
+    'ceph_versions',
+]
+
+GATHER_COMMANDS_JSON = [
+    'ceph_auth_list_--format_json-pretty',
+    'ceph_balancer_pool_ls_--format_json-pretty',
+    'ceph_balancer_status_--format_json-pretty',
+    'ceph_config-key_ls_--format_json-pretty',
+    'ceph_config_dump_--format_json-pretty',
+    'ceph_crash_ls_--format_json-pretty',
+    'ceph_crash_stat_--format_json-pretty',
+    'ceph_device_ls_--format_json-pretty',
+    'ceph_df_--format_json-pretty',
+    'ceph_fs_dump_--format_json-pretty',
+    'ceph_fs_ls_--format_json-pretty',
+    'ceph_fs_status_--format_json-pretty',
+    'ceph_fs_subvolume_ls_ocs-storagecluster-cephfilesystem_csi_--format_json-pretty',
+    'ceph_fs_subvolumegroup_ls_ocs-storagecluster-cephfilesystem_--format_json-pretty',
+    'ceph_health_detail_--format_json-pretty',
+    'ceph_mds_stat_--format_json-pretty',
+    'ceph_mgr_dump_--format_json-pretty',
+    'ceph_mgr_module_ls_--format_json-pretty',
+    'ceph_mgr_services_--format_json-pretty',
+    'ceph_mon_dump_--format_json-pretty',
+    'ceph_mon_stat_--format_json-pretty',
+    'ceph_osd_blacklist_ls_--format_json-pretty',
+    'ceph_osd_blocked-by_--format_json-pretty',
+    'ceph_osd_crush_class_ls_--format_json-pretty',
+    'ceph_osd_crush_dump_--format_json-pretty',
+    'ceph_osd_crush_rule_dump_--format_json-pretty',
+    'ceph_osd_crush_rule_ls_--format_json-pretty',
+    'ceph_osd_crush_show-tunables_--format_json-pretty',
+    'ceph_osd_crush_weight-set_dump_--format_json-pretty',
+    'ceph_osd_crush_weight-set_ls_--format_json-pretty',
+    'ceph_osd_df_--format_json-pretty',
+    'ceph_osd_df_tree_--format_json-pretty',
+    'ceph_osd_dump_--format_json-pretty',
+    'ceph_osd_getmaxosd_--format_json-pretty',
+    'ceph_osd_lspools_--format_json-pretty',
+    'ceph_osd_numa-status_--format_json-pretty',
+    'ceph_osd_perf_--format_json-pretty',
+    'ceph_osd_pool_ls_detail_--format_json-pretty',
+    'ceph_osd_stat_--format_json-pretty',
+    'ceph_osd_tree_--format_json-pretty',
+    'ceph_osd_utilization_--format_json-pretty',
+    'ceph_pg_dump_--format_json-pretty',
+    'ceph_pg_stat_--format_json-pretty',
+    'ceph_progress_--format_json-pretty',
+    'ceph_progress_json',
+    'ceph_progress_json_--format_json-pretty',
+    'ceph_quorum_status_--format_json-pretty',
+    'ceph_report_--format_json-pretty',
+    'ceph_service_dump_--format_json-pretty',
+    'ceph_status_--format_json-pretty',
+    'ceph_time-sync-status_--format_json-pretty',
+    'ceph_versions_--format_json-pretty',
+]
+
+
+GATHER_COMMANDS_OTHERS = [
+    'buildconfigs.yaml',
```

Comment:
Are you comparing those files are part of the must gather?  Are those files changing over the time?  For example in 4.5 must gather we collected only some of them and in 4.6 there are new one?  If so, how we will handle changes between versions?

Reply:
+1
also, the lists between versions sometimes changed (for example 4.4 to 4.5), and list-4.4 is not just subset of list-4.5

Reply:
@petr-balogh @AviadP 

Created list with common file names and list per-version.
for example:`GATHER_COMMANDS_CEPH =[]` `GATHER_COMMANDS_CEPH_4_5 = []` `GATHER_COMMANDS_CEPH _4_6=[]`